docs/integrations/saas-cloud/slack.md

```diff
@@ -188,7 +188,7 @@ The following table provides sample log messages for the different log types.
 }
 ```
 
-### Sample queries
+### Sample query
```

Comment:
@JV0812 we decided on keeping all of these headers consistent (plural). Easier to scan.

```suggestion
### Sample queries
```

Reply:
Thanks, I will update accordingly.